plugins/config-file.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-file               
+spec:
+  version: "0.0.6"      
+  platforms:
+  - selector:             
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Modulus/kubectl-cfg/archive/0.0.6.zip
+    sha256: "41952b7105e2a9f5f1b3fec306bc59c8e972578f9b5d543defeb970c5b5876d5"
+    files:
+    - from: "/kubectl-config-file*/*.sh" 
+      to: "."              
+    bin: "./kubectl-config-file.sh"  
+  shortDescription: List config files, and links chosen config file to ~/.kube/config without messing up your files.
```

Comment:
better alternative:

List available kubeconfigs and link them to ~/.kube/config.

?

Reply:
I have also changed this. I like your alternative better :)

Reply:
As for the `description` field, this one is still unchanged. Can you make sure to push the latest manifest version?